frontend/src/pages/Landing/LandingPage.tsx

```diff
@@ -572,31 +573,44 @@ function LandingPage() {
           >
             <div className={styles.EmailAddressContentDiv}>
               <Grid item>
-                <Typography className={styles.NewsletterRowHeader}>
-                  Engage in
-                  <br />
-                  Health Equity
-                </Typography>
+                <Hidden mdUp>
+                  <Typography className={styles.NewsletterRowHeaderSmall}>
+                    Engage in
+                    <br />
+                    Health Equity
+                  </Typography>
+                </Hidden>
+                <Hidden smDown>
+                  <Typography className={styles.NewsletterRowHeader}>
+                    Engage in
+                    <br />
+                    Health Equity
+                  </Typography>
+                </Hidden>
               </Grid>
               <Grid item>
                 <form
-                  className={styles.EmailAddressForm}
                   action="https://satcherinstitute.us11.list-manage.com/subscribe?u=6a52e908d61b03e0bbbd4e790&id=3ec1ba23cd&"
                   method="post"
                   target="_blank"
+                  className={styles.EmailTextWrapper}
```

Comment:
Could you change this class name to something like "NewsletterForm"?  Have been trying to move to a scheme where the HTML element name is always at the end of the class name